.github/workflows/push.yml

```diff
@@ -0,0 +1,42 @@
+name: Test my code
+
+on: push
+
+jobs:
+  lint:
+    name: Lint code
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Install deps
+      uses: docker://node:10-alpine
+      with:
+        entrypoint: npm
+        args: ci
+
+    - name: Lint
+      uses: docker://node:10-alpine
+      with:
+        entrypoint: npm
+        args: run lint
```

Comment:
@OmgImAlexis, ehat do you think changing this to

```yml
  lint:
    name: Lint code on Node v${{ matrix.node }}
    runs-on: ubuntu-latest
    strategy:
      matrix:
        node: ['10', '12', '13']

    steps:
      - uses: actions/checkout@master
      - uses: actions/setup-node@master
        with:
          node-version: ${{ matrix.node }}
      - run: npm ci
      - run: npm run lint
```

So it can be "tested" agains other Node versions?

Reply:
Does it really make sense to run the linter action against a matrix? If there were test cases to check the linter execution against a matrix it would be fine. But this is actually just ensuring the linter code does not have lint issues, so node versions should not matter, if they do, this would mean code issues and thus require proper test cases.